### PR TITLE
Add usage examples into README

### PR DESCRIPTION
**Pull request by 8W9aG**
* Also a little footnote over the design decision
regarding the delegate pattern over the block
pattern.

### CODE REVIEW

**Comment by 8W9aG** [comment]
@rastersize @dflems @jgavris

**Comment by jgavris** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -67,11 +67,55 @@ Next up, you need to add the framework to the Xcode project of your App. Lastly
...
+### Creating the SPTDataLoaderFactory
+Your app should ideally only create an SPTDataLoaderFactory once your user has logged in or if you require no authentication for your calls. The factory controls the authorisation of the different requests against authoriser objects that you construct.
+```objc
+id<SPTDataLoaderAuthoriser> oauthAuthoriser = [[SPTDataLoaderAuthoriserOAuth alloc] initWithDictionary:oauthTokenDictionary];
+self.oauthFactory = [self.service createDataLoaderFactoryWithAuthorisers:@[ oauthAuthoriser ]];
+```
+What we are doing here is using an implementation of an authoriser to funnel all the requests created by this factory into these authorisers.
+
+### Creating the SPTDataLoader
+Your app should create an SPTDataLoader object per view that wants to make requests (e.g. it is best not too share these between classes). This is so when your view model is deallocated the requests made by your view model will also cancel.
```
Comment (rastersize) [inline README.md]: Change `cancel.` :point_right: `cancelled.` (last word of the paragraph).

**Comment by rastersize** [comment]
:+1:

**Comment by 8W9aG** [comment]
Coverage change should be unaffected by the README

**Pull request merged**